src/Services/CalendarFetchService.cs

```diff
@@ -0,0 +1,94 @@
+﻿using Domain.Models;
+using Google.Apis.Calendar.v3;
+using Google.Apis.Services;
+using Persistence.Contexts;
+using Persistence.IRepository;
+
+namespace Services
+{
+    public class CalendarFetchService : ICalendarFetchService
+    {
+        private ApplicationDbContext _db;
+
+        string ApiKey;
+        string CalendarId = "sr.rs.official#holiday@group.v.calendar.google.com";
+
+        public CalendarFetchService(ApplicationDbContext db)
+        {
+            _db = db;
+            ApiKey = Environment.GetEnvironmentVariable("GOOGLE_CALENDAR_API_KEY")!;
+        }
```

Comment:
Apply refactoring that we did 1 on 1 and use `IUnitOfWork` instead of `ApplicationDbContext`

Reply:
solved